server/chat-commands/room-settings.ts

```diff
@@ -880,6 +880,38 @@ export const commands: Chat.ChatCommands = {
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
 		`Only users who are staff in a public room or global auth can make groupchats.`,
 	],
+	battlechat(target, room, user) {
+		room = this.requireRoom();
+		this.checkChat();
+		if (!room.battle) return this.errorReply("/battlechat must be used in a battle.");
+		const roomid = room.roomid.replace('battle-', 'battlechat-') as RoomID;
+		if (Rooms.get(roomid)) {
+			if (!this.runBroadcast()) return;
```

Comment:
For what it's worth: runBroadcast throws ErrorMessages, so you can just do:
```suggestion
			this.runBroadcast();
```